framework/templates/ecl-comments-with-form/ecl-comments-with-form.twig

```diff
@@ -0,0 +1,90 @@
+{#
+  Accepts:
+    - "image" (object): image to display on the left (optional)
+      - "src" (string): image's URL
+      - "alt" (string): alt text
+    - "author" (string): author name
+    - "date" (string): formatted date
+    - "title" (object): comment's title
+      - "href" (string): link to point to
+      - "text" (string): text to display
+    - "body" (string): comment's text
+    - "operations" (object): object of operations (reply, edit, delete)
+      - "delete|edit|reply" (object):
+        - href (string)
+        - title (string)
+        - visible (boolean)
+#}
+
+{# Internal properties #}
+
+{% set _css_class = 'ecl-row ecl-comments-with-form' %}
+{% set _extra_attributes = '' %}
+{% set _comments = comments %}
+
+{# Internal logic - Process properties #}
+
+{% if extra_class is defined %}
+  {% set _css_class = _css_class ~ ' ' ~ extra_class %}
+{% endif %}
+
+{% if extra_attributes is defined %}
+  {% for attr in extra_attributes %}
+    {% set _extra_attributes = _extra_attributes ~ ' ' ~ attr.name ~ '="' ~ attr.value ~ '"' %}
+  {% endfor %}
+{% endif %}
+
+<div class="{{ _css_class }}" {{ _extra_attributes }}>
```

Comment:
We don't need to encapsulate the code in a `div` (no class, no extra_attributes)